src/docs/JobPlanner/references/JobPlannerServiceReference.adoc

```diff
@@ -204,3 +204,74 @@ Given an event, a predefined action will be applied on the workflow execution.
 If an inclusion or exclusion calendar is not retrievable, it is blocking the <<_glossary_workflow,*Workflow*>> submission.
 An inclusion or exclusion calendar can become not retrievable if it cannot be downloaded from its URL and the
 <<_glossary_job_planner,*Job Planner*>> cache doesn't hold a copy.
+
+== Notifications
+
+The <<_glossary_job_planner,*Job Planner*>> can send notifications to users when particular events occur. The <<_glossary_notification_service,*Notification service*>> documentation can be found link:../admin/ProActiveAdminGuide.html#_notification_service[here].
+
+To start receiving notifications from the <<_glossary_job_planner,*Job Planner*>>, you must:
+
+- Configure your Job planner <<_glossary_notification_subscription,*Subscription*>> as it is described in the link:../user/ProActiveUserGuide.html#_service_subscriptions[Service subscriptions documentation] of the <<_glossary_notification_service,*Notification service*>>.
+- Inside the Job planner subscription, select from available events the ones you wish to be notified of.
+Available Job planner events with their description and example values are detailed in the link:../user/ProActiveUserGuide.html#_job_planner_notifications[Job Planner event types] table.
+
+== Job Planner Configuration
+
+<<_glossary_job_planner,*Job Planner*>> provides features to specify actions it will take regarding different situations it can encounter when submitting <<_glossary_workflow,*Workflows*>> according to a <<_glossary_calendars_definition,*Calendar*>>.
```

Comment:
Shouldn't be:
... provides features to specify actions : (or something similar)
1- it will take regarding different situations
2- it can encounter when submitting <<_glossary_workflow,*Workflows*>> according to a <<_glossary_calendars_definition,*Calendar*>>.

Otherwise the phrase seems unclear

Reply:
In my opinion your suggestion is less clear. 
The actions job planner takes depends on situations that happens when submitting worfklows.
So the two are linked and it seems confusing to me to separate it in two points like this.

Reply:
I understood what you meant, but you can add "that" before the second "it":
different situations that it can encounter when ...

I feel it will be easier to read.